src/languages/es.ts

```diff
@@ -891,6 +892,28 @@ export default {
             thatDidntMatch: 'Los 4 últimos dígitos de tu tarjeta no coinciden. Por favor, inténtalo de nuevo.',
         },
     },
+    // TODO: add translation
```

Comment:
@joekaufmanexpensify I am still missing some spanish translations that I could not find in the codebase. Could you get them for me, please? 😄